src/ecalc/libraries/neqsim/neqsim_ecalc_wrapper/thermo.py

```diff
@@ -260,6 +323,7 @@ def _remove_liquid(thermodynamic_system: ThermodynamicSystem) -> ThermodynamicSy
     def copy(self) -> NeqsimFluid:
         return NeqsimFluid(thermodynamic_system=self._thermodynamic_system.clone(), use_gerg=self._use_gerg)
 
+    @Capture.dump(dump_return_values=False, file_dump=Path(__file__).parent / "dump" / "neqsim-ph")
```

Comment:
Maybe a env-variable is a suitable way to enable / disable this?

Reply:
I agree, there is "a lot" that can be done to improve this, but this was currently thought to be for developers only to get the data out from neqsim, and I didnt want to add more than needed for that. It can be expanded in many ways later, for neqsim and other stuff imo, depending on need. From what I see it is now a "quick fix" to expand and tailor functionality based on what I have done, but did not think of doing that before we actually start gathering the data...